seeder/cli.ts

```diff
@@ -1,83 +1,77 @@
 import { parseArgs } from "@std/cli/parse-args";
 import { setBlobbyServerUrl, storage } from "@commontools/runner";
 import { setLLMUrl } from "@commontools/llm";
-import { processScenario } from "./processor.ts";
-import { type ExecutedScenario } from "./interfaces.ts";
-import { scenarios } from "./scenarios.ts";
-import { toolshedUrl } from "./env.ts";
-import { ensureReportDir, generateReport } from "./report.ts";
-import { browser, login } from "./jumble.ts";
 import { createSession, Identity } from "@commontools/identity";
 import { CharmManager } from "@commontools/charm";
+import { sleep } from "@commontools/utils/sleep";
```

Comment:
this one is funny since it is almost as long as the code required to define sleep 🤣